dll/win32/msv1_0/msv1_0.h

```diff
@@ -357,4 +357,10 @@ SystemFunction012(const BYTE *in,
                   const BYTE *key,
                   LPBYTE out);
 
+VOID
```

Comment:
This needs to go into a suitable NDK header.

Reply:
I moved this commit to a new PR #2357. Conversation will continue there.